Tools/glib/api_test_runner.py

```diff
@@ -317,7 +317,7 @@ def run_tests(self):
                 skipped_subtests = self._test_cases_to_skip(test)
                 number_of_total_tests += len(skipped_subtests if not subtests else set(skipped_subtests).intersection(subtests))
                 results = self._run_test(test, subtests, skipped_subtests)
-                if len(results) == 0:
+                if len(subtests) > 0 and len(results) == 0:
```

Comment:
I think this should be a separate change. You can land the gardening unreviewed and that makes it easier to revert this change if it causes any issue.

Reply:
Agreed. Please split this in two commits. You can land the gardening one unreviewed.
BTW, I think there is missing a bug number on the commit log.

Reply:
I split the patch in two and landed the gardening part:

- https://github.com/WebKit/WebKit/pull/33106